rskj-core/src/main/java/co/rsk/cli/tools/DbMigrate.java

```diff
@@ -183,12 +184,23 @@ private void migrate(DbMigrationInformation dbMigrationInformation) {
 
         logger.info("Migrating data source: {}", sourceKeyValueDataSource.getName());
 
-        sourceKeyValueDataSource.keys().forEach(sourceKey ->
-                targetKeyValueDataSource.put(
-                        sourceKey.getData(),
-                        sourceKeyValueDataSource.get(sourceKey.getData())
-                )
-        );
+        DataSourceKeyIterator iterator = sourceKeyValueDataSource.keyIterator();
+
+        while (iterator.hasNext()) {
```

Comment:
It would be nice to have a test to verify if both source and target datasources have the same content.

Reply:
Unit tests checks that